SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -1,13 +1,11 @@
 # Map travel
+<%(TOC)%>
 
-Before reading this topic, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).
-
-> Note: This topic is intended for advanced users only.
+This topic is intended for advanced users only. Before reading this page, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).
 
 <%(Callout type="warn" message="ServerTravel support has not yet been merged but will be coming very soon.")%>
 
-## ClientTravel
-`APlayerController::ClientTravel`
+## `APlayerController::ClientTravel`
```

Comment:
`APlayerController::ClientTravel`  Thought we agreed that titles shouldn't be in back-ticks. Looks strange in improbadoc.

Reply:
I wasn't privy to the convo - I was reverting to your original (had to fudge it for launch to make the link work).

Reply:
Fixed this and the other title which has the same formatting.

Reply:
Awesome thanks :D